httping V2.6: remove disabled status with new repo

iandol opened the pull request:
V2.5 had been disabled as the upstream repo disappeared. In fact the author moved it to github and it is available here (I confirmed with the original author this was his repo and is current). Trying to build the previous formula with the new repo code I ran into a few issues, for example on macOS 12.6 `useconds_t` was not properly imported and I reported this upstream(https://github.com/folkertvanheusden/HTTPing/issues/4) but doubt it will change according to the developer. My solution is to replace `useconds_t` with `unsigned int` in the formula as recommended by the Linux man page for `usleep`.

With that out of the way I also reported some other problems with the makefile which are now fixed upstream. 

This is building fine on my system. I also tested it on a Ubuntu machine and it seemed to also compile without issue.

Problems:

`brew audit --strict` tells me that: "_httping: * 4: col 3: Use versioned rather than branch tarballs for stable checksums._" -- however there is no versioning in that repo...

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

iandol commented:
Note: I know nothing about C, and I did ask for some advice here: https://github.com/orgs/Homebrew/discussions/3837

iandol commented:
Note: one potential problem — I didn't quite understand what `revision` was for and omitted it, but perhaps this needs to be added again?

SMillerDev commented on Formula/httping.rb:
```diff
@@ -1,11 +1,10 @@
-  license "GPL-2.0"
-  revision 2
+  homepage "https://github.com/folkertvanheusden/HTTPing"
+  url "https://github.com/folkertvanheusden/HTTPing/archive/refs/heads/master.zip"
```
Homebrew formulae need a tagged release upstream, this download is going to change (and possibly break) with every commit. Could you request a tagged release?

iandol replied:
OK, i'll open an issue and see where I get...

iandol replied:
https://github.com/folkertvanheusden/HTTPing/issues/6

iandol replied:
> @SMillerDev: Homebrew formulae need a tagged release upstream, this download is going to change (and possibly break) with every commit. Could you request a tagged release?

Done.

SMillerDev commented on Formula/httping.rb:
```diff
@@ -1,11 +1,10 @@
-  revision 2
+  homepage "https://github.com/folkertvanheusden/HTTPing"
+  url "https://github.com/folkertvanheusden/HTTPing/archive/refs/heads/master.zip"
+  sha256 "3fc401b5c1b853b04de4d546f6e49f6b907262aaae9c1cc9f173752217ebce48"
+  license "GPL-3.0-only"
+  version "2.6"
```
```suggestion
  version "2.6"
  sha256 "3fc401b5c1b853b04de4d546f6e49f6b907262aaae9c1cc9f173752217ebce48"
  license "GPL-3.0-only"
```

SMillerDev commented on Formula/httping.rb:
```diff
@@ -1,11 +1,10 @@
-  revision 2
+  homepage "https://github.com/folkertvanheusden/HTTPing"
+  url "https://github.com/folkertvanheusden/HTTPing/archive/refs/tags/v2.9.tar.gz"
+  version "2.9"
```
```suggestion
```
This can be omitted since it's clear from the url

SMillerDev commented on Formula/httping.rb:
```diff
@@ -19,18 +17,16 @@ class Httping < Formula
+    inreplace "utils.h", "useconds_t", "unsigned int"
     # Reported upstream, see: https://github.com/Homebrew/homebrew/pull/28653
-    inreplace %w[configure Makefile], "ncursesw", "ncurses"
+    inreplace %w[configure Makefile], "lncursesw", "lncurses"
```
Could you report this upstream too?

iandol replied:
https://github.com/folkertvanheusden/HTTPing/issues/7

SMillerDev replied:
Could you replace the commented link above with this link then

SMillerDev reviewed (commented):
One small thing and we're good to merge

iandol commented:
Hi, could I just take the time to say thank you @SMillerDev for your patience, help and all your hard work as a maintainer of homebrew!!!

SMillerDev reviewed (approved):
Thanks @iandol  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

SMillerDev commented:
Oh, could you squash your commits? CI doesn't like the merge commit you have in there.

iandol commented:
As I haven't squashed commits before I do want to make sure I do this correctly. So far I've just been using github's web browser interface on this pull request. I made the mistake of not branching (I thought it would auto-branch for me), and idly merging some changes from upstream without thinking they would pollute the pull request. I think i can `git rebase -i` my commits but what happens with that upstream merge?

SMillerDev commented:
I think you should be able to delete the merge commit if you do an interactive rebase.

iandol commented:
> I think you should be able to delete the merge commit if you do an interactive rebase.

OK, I think i deleted all the wrong merges and squashed my commits down to 1, lets see if it worked or I broke the universe...
